entgql/internal/todo/todo.graphql

```diff
@@ -90,6 +90,16 @@ input TodoWhereInput {
   and: [TodoWhereInput!]
   or: [TodoWhereInput!]
   
+  """id field predicates"""
+  id: ID
+  idNEQ: ID
+  idIn: [ID!]
+  idNotIn: [ID!]
+  idGT: ID
+  idGTE: ID
+  idLT: ID
+  idLTE: ID
```

Comment:
@giautm, @cliedeman - your thoughts here, please 🙏 

We usually use the Node API for querying by ID, so I'm not convinced that it's really needed, but please share your feedback.

Reply:
I have never found a usecase for this. I almost exclusively use the node api as the result of viewing a single objet from a url. E.g. www.example.com/person/12345

That being said. This api essentially exposes an almost complete sql query style so why would you exclude id? Maybe make it an opt in feature?

I do plan to eventually add an `Includes` annotation to pair with the `Skip` annotation and allow the user to configure if inclusion or exclusion is the default for types, edges and fields.

Reply:
An example why this would be needed: filtering a relation

```graphql
query ALL_PRODUCTS {
  products(orderBy: { direction: DESC, field: NAME }, where: { hasCategoriesWith: { id: 5 }}) {
    totalCount
    edges {
      cursor
      node {
        id
      }
    }
  }
}

```

Reply:
> An example why this would be needed: filtering a relation


It make sence to me. But I think we should limit ops to: EQ/NotEQ/In/NotIn?

Reply:
makes sense! 

But it runs over the ID ops that already here*: I'm not sure how GT/GTE/... on IDs would make sense, but i mean they are available already...

*  https://github.com/ent/contrib/blob/master/entgql/internal/todo/ent/todo/where.go#L28-L108

Reply:
I suggest leaving it as it is for now and add an option to configure it using custom annotation/option in the future.